src/runtime/cuda/cuda_device_api.cc

```diff
@@ -283,7 +285,7 @@ class GPUTimerNode : public TimerNode {
 
 TVM_REGISTER_OBJECT_TYPE(GPUTimerNode);
 
-TVM_REGISTER_GLOBAL("profiling.timer.gpu").set_body_typed([](Device dev) {
+TVM_REGISTER_GLOBAL("profiling.timer.cuda").set_body_typed([](Device dev) {
```

Comment:
So it was looking for `profiling.timer.cuda` this entire time? Why wouldn't profiling crash when it could not find it?

Reply:
There's a default timer that is used. Maybe we should warn if the timer can not be found?

Reply:
I think this would prevent future errors like this from occuring. If it is easy to add in this PR

Reply:
done